src/finn/util/basic.py

```diff
@@ -216,3 +216,21 @@ def is_exe(fpath):
                 return exe_file
 
     return None
+
+
+def find_next_power_of_2(n):
```

Comment:
Please also provide docstring here for documentation.

Reply:
Added - see https://github.com/Xilinx/finn/pull/687/commits/0d99b6c8ed358b2feea41cc8af242d40b30c8d97